applications/feedback_sentiment_analyzer/cdk/lib/functions.ts

```diff
@@ -3,11 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { resolve } from "path";
```

Comment:
Throughout the typescript files, your formatter is going against the grain. I'm not sure we've documented it explicitly (and we should), but I think @cpyle0819 and I are using prettier's default configuration.

Reply:
I installed Prettier and got it working.